cSpell.json

```diff
@@ -0,0 +1,84 @@
+{
+	"version": "0.2",
+	"language": "en",
+	"words": [
```

Comment:
I don't think we can commit to maintaining a list like this. We appreciate the typo fixes, but we can't afford an include list with the dictionary words and a dependency to cspell at this point.